cli/jobs/pipelines-with-components/rai_pipeline_adult_analyse/components/component_gather_rai_insights.yaml

```diff
@@ -0,0 +1,40 @@
+$schema: http://azureml/sdk-2-0/CommandComponent.json
```

Comment:
let's update these schema to clickable urls

Reply:
updated